queue_processors/queue_processor/queue_listener.py

```diff
@@ -63,10 +73,8 @@ def on_message(self, headers, message):
             self.logger.info("Destination: %s Priority: %s", destination, priority)
             # Load the JSON message and header into dictionaries
             try:
-                if not isinstance(message, Message):
```

Comment:
It is not possible for the listener to recieve a message object so this if check was always `True`